awscli/help.py

```diff
@@ -310,3 +356,135 @@ def event_class(self):
     @property
     def name(self):
         return self._name
+
+
+class TopicListerCommand(HelpCommand):
+    EventHandlerClass = TopicListerDocumentEventHandler
+
+    DESCRIPTION = (
+        'This is the AWS CLI Topic Guide. It gives access to a set '
+        'of topics that provide a deeper understanding of the CLI. To access '
+        'the list of topics from the command line, run ``aws help topics``. '
+        'To access a specific topic from the command line, run '
+        '``aws help [topicname]``, where ``topicname`` is the name of the '
+        'topic as it appears in the output from ``aws help topics``.')
+
+    def __init__(self, session, topic_tag_db):
+        super(TopicListerCommand, self).__init__(session, None, {}, {})
+        self._topic_tag_db = topic_tag_db
+        self._categories = None
+        self._entries = None
+        self._related_items = [
+            'AWS CLI Reference Guide (`aws help <../reference/index.html>`_)']
+
+    @property
+    def event_class(self):
+        return 'topics'
+
+    @property
+    def name(self):
+        return 'topics'
+
+    @property
+    def title(self):
+        return 'AWS CLI Topic Guide'
+
+    @property
+    def description(self):
+        return self.DESCRIPTION
+
+    @property
+    def categories(self):
```

Comment:
Same thing here.  My initial reaction is that the layering is off.  The help commands previously didn't have any of these domain-specific attributes.  This again also provides an inconsistent interface to the HelpCommand objects.